### DISCUSSION TITLE
brew list command

### DISCUSSION THREAD

**Discussion by vasanthsadasivan** [Homebrew/discussions#3357] [Everyday usage]
Wondering if anyone is encountering the same issue (not problem). When I use brew list (and I think other brew commands too) it takes an abnormal amount of time for a result, for brew list its like 20 seconds. Any suggestions?

**Comment by apainintheneck** [comment]
Is this a new behavior or has it always been this slow for you?

**Comment by vasanthsadasivan** [comment]
Prolly not from the get go but in my case, brew has spanned a few laptops, so system hasn't been the same (all Apple Macbook Air). On my latest Mac it has been same delay for a while now. I mean for me, listing installed packages seems pretty trivial  and minimum log access to take 20 seconds to complete. Appreciate your time. Thank you.

**Comment by vasanthsadasivan** [comment]
Haven't searched, but is there a freeze option for brew to stamp a file with all installed casks/pkgs kinda like a requirements file in python to load dependant libs? Is there options to auto load this if I have to re-install? Appreciate your time. Thank you.

**Reply by SMillerDev** [reply]
That's essentially `brew bundle`

**Reply by vasanthsadasivan** [reply]
Thank you
